### ISSUE TITLE
Failed to patch any app

### ISSUE THREAD

**Issue by Zerogoki00** [reisxd/revanced-builder#679]
**Describe the bug**
Failed to patch any app

**To Reproduce**
Try to patch any app with any patches

```
[builder]
An error occured:
Error: EBUSY: resource busy or locked, unlink '\\?\D:\Progs\apk\revanced-cache\1.apk'
at unlinkSync (node:fs:1780:3)
at _unlinkSync (node:internal/fs/rimraf:214:14)
at rimrafSync (node:internal/fs/rimraf:195:7)<br>
at node:internal/fs/rimraf:253:9
at Array.forEach (<anonymous>)
at _rmdirSync (node:internal/fs/rimraf:250:7)
at rimrafSync (node:internal/fs/rimraf:193:7)
at rmSync (node:fs:1274:10)
at afterBuild (C:\snapshot\revanced-builder\wsEvents\patchApp.js:28:3)
at Socket.<anonymous> (C:\snapshot\revanced-builder\wsEvents\patchApp.js:186:53)
Please report this bug here: https://github.com/reisxd/revanced-builder/issues.
```

**Comment by Zerogoki00** [comment]
I deleted `revanced` and `revanced-cache` folders and rebooted to start from clean install
But I still get the same error

**Comment by Egglestron** [comment]
I installed the `revanced-signed.apk` from the `revanced-cache` folder and it seems to work! Not ideal buy hey, better than nothing.

**Comment by schumi2004** [comment]
@Egglestron can you elaborate on that? What do yoy mean with installed revanced-signed.apk, on your phone?
Like @Zerogoki00 it's unable to patch any app and don't see yet what installing revanced-signed.apk would help on that.

**Comment by GieltjE** [comment]
> 

This doesn't work for me, always get a conflict.

**Comment by shrihanDev** [comment]
Closed via c2707da6996a7b7f3a172a31e6888660a8f839f4
